distribution/lib/Standard/Base/0.0.0-dev/src/Network/URI.enso

```diff
@@ -64,6 +64,8 @@ type URI
                 truncated = if message.is_nothing || message.length > 100 then "Invalid URI '" + uri.to_display_text + "'" else
                     "URI syntax error: " + message
                 Error.throw (Syntax_Error.Error truncated)
+        if raw_uri.scheme == Nothing then URI.parse ("https://" + uri) else
+            raw_uri
```

Comment:
Note: this will change behaviour for URIs like `//a` which I guess are valid.

Before: `URI.parse "//a" . host == "a"`

After: `URI.parse "//a" . host == Nothing`